modules/ROOT/pages/visualizer-app-network.adoc

```diff
@@ -67,7 +71,7 @@ The *Details* tab contains:
 
 The *Monitoring* tab contains:
 
-* *Metrics summary*: Provides monitoring metrics for the selected node, including Errors count, avg. response time, avg. throughput, avg. CPU consumption, and avg. memory utilization.
+* *Metrics summary*: Provides monitoring metrics for the selected node, including Avg response time, Avg throughput, Failures, Avg CPU utilization, and Avg memory utilization. This shows either inbound or outbound values for Avg response time, Avg throughput, and Failures depending on what you’ve selected in the *Metric* card.
```

Comment:
GLOBALLY limit list items or steps to one sentence or fragment. If additional info is necessary, move it to a plain text para below that list item:
```suggestion
* *Metrics summary*
Provides monitoring metrics for the selected node, including average response time, average throughput, failures, average CPU utilization, and average memory utilization. This shows either inbound or outbound values for Avg response time, Avg throughput, and Failures, depending on what you’ve selected in the *Metric* card.
```
My approach here is the spell out the type of metric but use UI labels when referring to what is shown in the UI. Might be too granular.